packages/nx/src/devkit-exports.ts

```diff
@@ -219,7 +219,8 @@ export {
 /**
  * @category Utils
  */
-export { Hash, Hasher } from './hasher/hasher';
+export { Hash, TaskHasher } from './hasher/task-hasher';
```

Comment:
We should do something like this to avoid breaking existing plugins that use the hasher.

```suggestion
export { Hash, TaskHasher, TaskHasher as Hasher } from './hasher/task-hasher';
```